src/pages/Jobs/ScheduleAgents/index.js

```diff
@@ -16,18 +16,231 @@
  51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { Spinner } from "react-bootstrap";
+import CommonFields from "../../../components/Upload/CommonFields";
+import InputContainer from "../../../components/Widgets/Input";
+import Alert from "../../../components/Widgets/Alert";
+import Button from "../../../components/Widgets/Button";
+import { getAllFolders } from "../../../services/folders";
+import { scheduleJobs } from "../../../services/upload";
+import { getBrowseData } from "../../../services/browse";
 
 const ScheduleAgents = () => {
+  const initialScheduleAnalysisData = {
+    folderId: 1,
+    uploadId: 1,
+  };
+  const initialScanFileData = {
+    analysis: {
+      bucket: true,
```

Comment:
Please read buckets as well from UI.

Reply:
@GMishx Updated it.

Reply:
Now bucket is checked by default. Also, in current UI it is the first item, please shift accordingly.

Reply:
@GMishx Updated it